### PR TITLE
Fixes #7530: Several systemd warnings in install on RHEL/CentOS 7

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7530

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -500,7 +500,7 @@ then
   /usr/sbin/mkitab "rudder-agent:23456789:once:/usr/bin/startsrc -s rudder-agent"
   # No need to tell init to re-read /etc/inittab, it does it automatically every 60 seconds
 %else
-  RUDDER_AGENT_INIT_ENABLED=$(LANG=C chkconfig --list | grep -Ec "rudder-agent.*on")
+  RUDDER_AGENT_INIT_ENABLED=$(LANG=C chkconfig --list 2>/dev/null | grep -Ec "rudder-agent.*on")
```
Comment (jooooooon) [inline rudder-agent/SPECS/rudder-agent.spec]: OK, that works...

**Comment by jooooooon** [comment]
This fixes one of the problems in the ticket, but there are several left:

1 and 2 (in rudder-webapp):

```
==> server_centos7: INFO: Setting Apache HTTPd as a boot service...Note: Forwarding request to 'systemctl enable httpd.service'.
==> server_centos7: ln -s '/usr/lib/systemd/system/httpd.service' '/etc/systemd/system/multi-user.target.wants/httpd.service'
==> server_centos7:  Done
==> server_centos7: INFO: Restarting syslog...Redirecting to /bin/systemctl restart  rsyslog.service
==> server_centos7:  Done
```

3 (in rudder-inventory-endpoint):

```
==> server_centos7: Restarting syslogd ...
==> server_centos7: Redirecting to /bin/systemctl restart  rsyslog.service
```

**Comment by peckpeck** [comment]
Those are normal informations printed by systemd

**Comment by jooooooon** [comment]
Maybe, but they mess up our output and are not necessary. We should simply call the systemd commands instead.

**Comment by peckpeck** [comment]
The "forwarding" part can be removed, but the "ls" part cannot.

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
ping @jooooooon

**Comment by jooooooon** [comment]
Note for self: please review this

**Comment by jooooooon** [comment]
Looks good to me! Thanks for the new version, and sorry I missed this update when you made it, months ago.

**Pull request merged**